web/modules/custom/rdf_serialization/src/Normalizer/RdfEntityNormalizer.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Drupal\rdf_serialization\Normalizer;
+
+use Drupal\rdf_export\RdfSerializer;
+use Drupal\serialization\Normalizer\FieldableEntityNormalizerTrait;
+
+/**
+ * Converts the Drupal entity object structure to a HAL array structure.
+ */
+class RdfEntityNormalizer extends NormalizerBase {
+
+  use FieldableEntityNormalizerTrait;
+
+  /**
+   * The interface or class that this Normalizer supports.
+   *
+   * @var string
+   */
+  protected $supportedInterfaceOrClass = 'Drupal\rdf_entity\RdfInterface';
```

Comment:
Let's not hardcode the namespaced interface name. Use `RdfInterface::class` instead.